project/Dependencies.scala

```diff
@@ -2,11 +2,16 @@ import sbt._
 
 object Dependencies {
   val x = List(1) // scalafix:ok
-  def scalafixVersion: String = "0.14.2"
+
+  // keep this as low as possible, to allow bumping sbt-scalafix without scalafix-interfaces
+  def scalafixInterfacesVersion: String =
+    "0.14.2+48-9b6e03ac-SNAPSHOT" // scala-steward:off
```

Comment:
🛑 this must be updated to a stable version before merging, or at least tagging a release